work/worker.go

```diff
@@ -941,7 +946,7 @@ func (env *Task) commitBundleTransaction(bundle *builder.Bundle, bc BlockChain,
 			if err == nil {
 				err = kerrors.ErrRevertedBundleByVmErr
 			}
-			return err, tx, nil
+			return err, nil, nil
```

Comment:
I think we need to return `tx`, not nil.

```suggestion
			return err, tx, nil
```